scripts/deploy_protocol.py

```diff
@@ -1,39 +1,55 @@
-import sys
-import asyncio
+import os
+import logging
+import json
+from asyncio import run
 from starknet_py.net.account.account_client import (AccountClient)
 from starknet_py.net.signer.stark_curve_signer import KeyPair
 from starknet_py.contract import Contract
 from starknet_py.net.models import StarknetChainId
 from starkware.crypto.signature.signature import private_to_stark_key
 from starknet_py.net.gateway_client import GatewayClient
-from utils import deployContract
+from utils import declare_and_deploy_contract, declare_contract
 from pathlib import Path
+from dotenv import load_dotenv
+
+#Logging conf, just want to print results to console
+logging.basicConfig(level = logging.INFO)
+
+# Address of the ETH token on mainnet,testnet1 and testnet2
+ETH_ADDRESS = 2087021424722619777119509474943472645767659996348769578120564519014510906823
+# Set a high max fee for the deployments
+MAX_FEE = int(1e16)
+
+# Loading .env file
+dotenv_path = os.path.join('./', '.env')
+load_dotenv(dotenv_path)
+
+# Get env variables
+private_key = int(os.environ.get("PRIVATE_KEY"))
+account_address = int(os.environ.get("ACCOUNT_ADDRESS"),16)
```

Comment:
looks like you don't have a linter that runs locally ;) (missing a space between `,` and `16`)